src/containers/HomeScreenContainer.js

```diff
@@ -1,18 +1,40 @@
 import {connect} from 'react-redux';
 import Home from '../screens/Home';
-import {setCategories, setCurrentCategory, setPhrases} from '../redux/actions';
-import {categoriesRoot, nativeLanguageRoot} from '../redux/selectors';
+import {
+  setCategories,
+  setCurrentCategory,
+  setPhrases,
+  setLearntPhrases,
+  synchronizeStorageToRedux,
+  setSeenPhrases,
+  addSeenPhrase,
+} from '../redux/actions';
+
+import {
+  categoriesRoot,
+  nativeLanguageRoot,
+  learntPhrases,
+  seenPhrases,
+  leftPhrases,
```

Comment:
If this is to get the phrases that are left in a category, consider changing it to
```suggestion
  remainingPhrases,
```